Fix documentation for LDAP Groups (#3614)
Icinga/icingaweb2: doc/05-Authentication.md

-------------------------|-----------------------------------------------
backend                  | **Required.** Specifies the backend type. Can be set to `ldap`, `msldap`.
resource                 | **Required.** The name of the LDAP resource defined in [resources.ini](04-Resources.md#resources).

> base_dn                  | **Optional.** LDAP base dn for groups.

Reviewer: `user_class` and `user_name_attribute` are valid configuration options here and shouldn't be removed. Though, `user_class` is not quite correct. The description is missing the `msldap` default and the `base_dn` may also include a better description:

```suggestion
user\_class              | **Optional.** LDAP user class. Defaults to `inetOrgPerson` with `msldap` and `user` with `ldap`.
user\_name\_attribute    | **Optional.** LDAP attribute which contains the username. Defaults to `sAMAccountName` with `msldap` and `uid` with `ldap`.
base_dn                  | **Optional.** LDAP base dn for groups. Leave empty to select all groups available using the specified resource.
```

group\_class             | **Optional.** LDAP group class. Defaults to `group`.

> group\_member\_attribute | **Optional.** LDAP attribute which contains the username. Defaults to `sAMAccountName` with `msldap` and `member` with `ldap`.

Reviewer: It's `member` for both backend types and the description is wrong:

```suggestion
group\_member\_attribute | **Optional.** LDAP attribute where a group's members are stored. Defaults to `member`.
```

group\_name\_attribute   | **Optional.** LDAP attribute which contains the groupname. Defaults to `sAMAccountName` with `msldap` and `gid` with `ldap`.
group\_filter            | **Optional.** LDAP group search filter. Requires `group_class` and `group_name_attribute`.

> nested\_group\_search    | **Optional.** Enable nested group search in Active Directory based on the user. Defaults to `0`. Only available with `backend` type `msldap`.

Reviewer: Additionally, please add the missing options `user_base_dn` and `domain` as well. In case of `domain` please refer to the `Domain-aware Authentication` section below.
